core-blocks/image/index.js

```diff
@@ -165,6 +170,10 @@ export const settings = {
 	save( { attributes } ) {
 		const { url, alt, caption, align, href, width, height, id } = attributes;
 
+		const classes = classnames( align ? `align${ align }` : null, {
```

Comment:
Minor: Good use case for `classnames` object format is to inverse this, so you don't need to assign a fallback `null` value to be omitted:

```js
const classes = classnames( {
	[ `align${ align }` ]: align,
	// ...
} );
```